docs/guide.md

```diff
@@ -300,6 +303,15 @@ If you're working on web services and want to access them locally, code-server
 can proxy to any port using either a subdomain or a subpath, allowing you to
 securely access these services using code-server's built-in authentication.
 
+By default, ports running on the same machine as code-server can be accessed at
+{current_url}/proxy/{port}. For instance, if you have code-server running on
+localhost:8080 and a Python server running on localhost:8000, you could access
+it via http://localhost:8080/proxy/8000
+
+You can also override the URL scheme for the proxy using the `VSCODE_PROXY_URI`
+environment variable. `VSCODE_PROXY_URI=https://{{port}}.kyle.dev` would forward
+an application running on localhost:3000 to https://3000.kyle.dev
```

Comment:
To me this makes it sound like `VSCODE_PROXY_URI` will forward your applications automatically but this is essentially a display-only change so you need to have your own proxy hosted at whatever you set this.

What about a new section like:

```
### Using your own proxy

You can make extensions and the ports panel use your own proxy by setting `VSCODE_PROXY_URI`.  For example if you set `VSCODE_PROXY_URI=https://{{port}}.kyle.dev` when an application is detected running on port 3000 of the same machine code-server is running on the ports panel will create a link to https://3000.kyle.dev instead of pointing to the built-in proxy.
```

Reply:
I like how you worded this better. Good idea!